docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -207,27 +222,32 @@ $group_info = $group->get_info();
 The customer group ID is not available anywhere on the client side.
 
 ## Custom CSS
+
 To style BigCommerce for WordPress elements with custom CSS, add your CSS to your theme’s stylesheet rather than editing the plugin stylesheets directly. Your theme’s CSS will have specificity over styles applied by the plugin and will override the default styles.
 
 For example, you might want to change the Add-To-Cart button from blue to orange by adding the following CSS to your theme’s stylesheet:
 
 ```css
 button.bc-btn.bc-btn--form-submit.bc-btn--add_to_cart {
-    background-color: #FF9C33;
+  background-color: #ff9c33;
 }
 ```
+
 ### Opting Out of BigCommerce Styles
+
 If you wish to disable the built-in plugin styles entirely, you have the option to do so. In the WordPress theme customizer, navigate to BigCommerce > Colors & Theme and select Disable Plugin Styles from the CSS dropdown menu.
 
 ## Hooks
-Hooks are access points during the WordPress execution process where a developer can insert custom code. Hooks consist of two types: actions and filters. Both allow developers to execute custom code during the WordPress lifecycle. The difference lies in whether the function returns a value: 
 
-- **Actions** execute a function with no output. Even if a value was returned it would be ignored. 
+Hooks are access points during the WordPress execution process where a developer can insert custom code. Hooks consist of two types: actions and filters. Both allow developers to execute custom code during the WordPress lifecycle. The difference lies in whether the function returns a value:
+
+- **Actions** execute a function with no output. Even if a value was returned it would be ignored.
 - **Filters** modify a variable and return a value, which is the modified version of the original variable.
 
 BigCommerce for WordPress provides over 100 hooks that you can use to extend and customize the plugin. For a comprehensive, searchable list of all available hooks, visit our [Code Reference](https://bigcommerce.moderntribe.qa/reference/hooks/).
 
 ### Architectural Guidelines
+
 All actions and filters called by the plugin begin with the `bigcommerce/` prefix (e.g., `bigcommerce/init`). If there is a dynamic component to the hook, it should be preceded by an equal sign (e.g., `bigcommerce/template=' . $template . '/path`).

```

Comment:
All actions and filters called by the plugin begin with the `bigcommerce/` prefix (e.g., `bigcommerce/init`). If there is a dynamic component to the hook, precede it with an equal sign (e.g., `bigcommerce/template=' . $template . '/path`).